src/firefly/js/api/ApiUtil.js

```diff
@@ -35,8 +35,8 @@ export const isDebug = () => get(window, 'firefly.debug', false);
  * @param {String|Error} msg any number of messages
  */
 export function debug(...msg) {
-    if (!isEmpty(msg) && isDebug()) {
-        logError(...msg);
+    if (isDebug() && !isEmpty(msg)) {
+        logErrorWithPrefix('Firefly:', ...msg);
```

Comment:
this is more of a question.  is this how msg suppose to be passed?
you cannot just do msg instead of ...msg?